scripts/spawn_sdf_test.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python
+from ed_object_models import spawn_sdf_in_gazebo
+
+if __name__ == "__main__":
+    # This spawns 7 coke cans, similar to fast_simulator_data/worlds/robotics_testlabs/robotics_testlabs.py
+    spawn_sdf_in_gazebo.from_yaml('/sdf_test/objects_list_test.yaml')
```

Comment:
This looks like an absolute path to me. Which will not work. Best solution is to use rospack to find the package. Or have the path as an argument.

Reply:
I changed the functionally of the function, which now also has a package name as input (default is fast_simulator_data).